driver-core/src/main/com/mongodb/client/model/expressions/Expressions.java

```diff
@@ -177,6 +177,19 @@ public static <R extends DocumentExpression> R current() {
                 .assertImplementsAllExpressions();
     }
 
+    /**
+     * Returns the "current" value as a {@linkplain MapExpression map} value.
+     * The "current" value is the top-level document currently being processed
+     * in the aggregation pipeline stage.
+     *
+     * @return the resulting value
+     * @param <R> the type of the resulting value
+     */
```

Comment:
[optional]

This text is different from that in https://github.com/mongodb/mongo-java-driver/pull/1059.